roles/adoption_osp_deploy/tasks/login_registries.yml

```diff
@@ -0,0 +1,50 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Activate red hat subscription
+  when:
+    - cifmw_adoption_osp_deploy_rhsm_org is defined
+    - cifmw_adoption_osp_deploy_rhsm_key is defined
+  become: true
+  no_log: true
+  ansible.builtin.command: >-
+    subscription-manager register --force
+    --org "{{ cifmw_adoption_osp_deploy_rhsm_org }}"
+    --activationkey "{{ cifmw_adoption_osp_deploy_rhsm_key }}"
+
+- name: Login in container registry
+  when:
+    - cifmw_adoption_osp_deploy_container_user is defined
+    - cifmw_adoption_osp_deploy_container_password is defined
+    - cifmw_adoption_osp_deploy_container_registry is defined
+  block:
+    - name: Install podman for container registry login
```

Comment:
good point, I can use the `podman` role for that, will make the change

Reply:
@lewisdenny I tried using the `podman` role, but that uses the `package` module instead of `dnf` and I'm hitting this issue https://github.com/ansible/ansible/issues/82598 I'm not sure what is the problem, but tbh I don't think it's worth it to try and find out the problem

Reply:
No you are right, not worth it!